app/static/js/displayFilesMacro.js

```diff
@@ -1,27 +1,28 @@
 $(document).ready(function(){
     $("a.fileName").tooltip();
 
-    $(".removeAttachment").on("click", function(){
-        let fileId = $(this).data("id");
-        let deleteLink = $(this).data("delete-url");
-        let fileData = {
-            fileId: fileId,
-            databaseId: $(this).data('database-id')
-        };
+    // $(".removeAttachment").on("click", function(e){
+    //     e.preventDefault();
+    //     let fileId = $(this).data("id");
+    //     let deleteLink = $(this).data("delete-url");
+    //     let fileData = {
+    //         fileId: fileId,
+    //         databaseId: $(this).data('database-id')
+    //     };
 
-        $.ajax({
-            type: "POST",
-            url: deleteLink,
-            data: fileData,
-            success: function(){
-                msgFlash("Attachment removed successfully", "success");
-                $("#attachment_" + fileId).remove();
-            },
-            error: function(error){
-                msgFlash(error);
-            }
-        });
-    });
+    //     $.ajax({
+    //         type: "POST",
+    //         url: deleteLink,
+    //         data: fileData,
+    //         success: function(){
+    //             msgFlash("Attachment removed successfully", "success");
+    //             $("#attachment_" + fileId).remove();
+    //         },
+    //         error: function(error){
+    //             msgFlash(error);
+    //         }
+    //     });
+    // });
 
```

Comment:
We still need this code in other places.

Reply:
Uncommented the code.